client/state/notices/utils.js

```diff
@@ -3,10 +3,10 @@
  */
 import { successNotice, errorNotice } from 'state/notices/actions';
 
-export function dispatchSuccess( message ) {
-	return ( dispatch ) => dispatch( successNotice( message ) );
+export function dispatchSuccess( ...args ) {
+	return ( dispatch ) => dispatch( successNotice.apply( null, args ) );
```

Comment:
You could also do ES6 argument spread, which is slightly more compact:

```js
dispatch( successNotice( ...args ) )
```

Potentially you could even drop `args` and use the [arguments object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments), though this may be less readable:

```js
export function dispatchSuccess() {
	return ( dispatch ) => dispatch( successNotice( ...arguments ) );
}
```

(Takes advantage of arrow functions [not binding `arguments`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#No_binding_of_arguments))